.github/workflows/initiate_test_deployment.yml

```diff
@@ -0,0 +1,22 @@
+name: Run prefect flow
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        default: "Deploy test"
+jobs:
+  run-using-flow-path:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Perform prefect login
+        uses: sp1thas/prefect-cli-action@main
+        with:
+          command: prefect auth login --key ${{ secrets.PREFECT_API_KEY }}
+      - name: Deploy Test Commission Flow 
+        uses: sp1thas/prefect-cli-action@main
```

Comment:
do you need to specify a prefect version anywhere in here?

Reply:
Another very good question! I may, but I may not, as this simply is being used to make an API call to prefect to cause a flow to run, and not to compile and register one. I'll report back as soon as I'm able to test this. If so, I may have to do a little bit more of a custom github image to pin prefect to a version, but I'm hopeful that this packaged up prefect client does the trick. Thanks for asking - I'll check for sure.